packages/minimongo/modify.js

```diff
@@ -185,20 +185,20 @@ var MODIFIERS = {
   $currentDate: function (target, field, arg) {
     if (typeof arg === "object" && arg.hasOwnProperty("$type")) {
        if (arg.$type !== "date") {
-         throw MinimongoError("Minimongo does currently only support the date type in $currentDate modifiers");
+         throw MinimongoError("Minimongo does currently only support the date type in $currentDate modifiers: " + field);
```

Comment:
@benjamn I thought of doing that. Actually, I thought of converting it into a Meteor Error format (code, reason, message, details) but then I thought it was too much work just for adding the field name. I still think so. :). IMO, adding one parameter does not make much sense, but conforming it to the `Meteor Error` class (with a new errorType `Mongo.Error`) definitely does. It will also help with displaying the errors as we will be able to use err.message for any kind of error. I believe that minimongo is the only place where we get a vanilla javascript `Error`.
If you have any ideas as to some other info that we can include in the error, then I will try to incorporate them altogether and raise a PR whenever I get some time.